webhook_to_fedora_messaging/endpoints/parser/parser.py

```diff
@@ -0,0 +1,63 @@
+
+from  flask import Request
+from werkzeug.datastructures.headers import EnvironHeaders
+import hashlib
+import hmac
+from webhook_to_fedora_messaging_messages.github.github import GithubMessageV1
+from ..util import fas_by_github
+
+
+
+def msg_parser(service_type: str, request: Request):
```

Comment:
Same comment about passing `request` around, valid for the whole file. Just use `flask.request`.